lib/cluster.js

```diff
@@ -293,28 +293,42 @@ function masterInit() {
     cluster.emit('setup', settings);
   }
 
-  var debugPortOffset = 1;
+  var debugPortOffset = 0;
+  var debugArgRegex = /^(--inspect|--debug|--debug-(brk|port))(=(.+:)?(\d+))?$/;
```

Comment:
You can use a non-capturing match (`(?:=)`) to reduce the index count a little.  Makes the code below a little easier to comprehend.